Makefile

```diff
@@ -46,6 +46,7 @@ install: gopath
 	$(GOPATH)/bin/mixer-completion zsh --path $(DESTDIR)/usr/share/zsh/site-functions/_mixer
 	install -m 00755 pack-maker.sh $(DESTDIR)/usr/bin/mixer-pack-maker.sh
 	install -m 00755 superpack-maker.sh $(DESTDIR)/usr/bin/mixer-superpack-maker.sh
+	install -m 00644 docs/* $(DESTDIR)/usr/share/man/man1/
```

Comment:
This will install the RST files into those folders too?

Reply:
And won't work if you add `.5` section man pages, but for now it's ok.

Reply:
> This will install the RST files into those folders too?

Ooops good point.

> And won't work if you add .5 section man pages, but for now it's ok.

Yeah I figured I'd cross that bridge when I get there.

Reply:
> This will install the RST files into those folders too?

fixed.

Reply:
use `$(MANPAGES)` here

Reply:
ok, *that's* done now.